kotlin-styled-next/src/test/kotlin/test/RemoveCssTest.kt

```diff
@@ -0,0 +1,168 @@
+package test
+
+import Component
+import TestScope
+import createDOMElement
+import kotlinx.css.color
+import kotlinx.css.paddingLeft
+import kotlinx.css.paddingRight
+import kotlinx.css.px
+import kotlinx.dom.clear
+import org.w3c.dom.Element
+import org.w3c.dom.HTMLElement
+import react.Props
+import react.fc
+import runTest
+import styled.css
+import styled.styledDiv
+import styled.styledSpan
+import unmount
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertNotNull
+
+class RemoveCssTest : TestBase() {
```

Comment:
There should be tests about removing nested rules. Also it looks like there should be the way how to unmount "static" StyleSheets and globally injected styled. And test for them too.